### DISCUSSION TITLE
AzuraCast v0.17.7 Stable Does not connect Live sources

### DISCUSSION THREAD

**Discussion by omarelifaz** [AzuraCast/AzuraCast#6208] [General]
Greetings and Blessings, please I need help, I have the version of AzuraCast v0.17.7 Stable • Docker • PHP 8.2, the problem is that I cannot connect live programming, the panel shows me station off, but the autodj is playing normally. with icecast and shoucast. Thank you...

**Comment by BusterNeece** [comment]
I'm not sure what would be causing that in this situation. If you're still encountering this problem, could you create a new Issue (instead of Discussion) with any relevant logs that might help? One log of importance is the Liquidsoap log, which you can view from the "Logs" page on your station profile.
